app/components/Texting/Texting.scss

```diff
@@ -0,0 +1,175 @@
+@import '../../styles/utils/_helpers.scss';
+
+.overlay {
+  position: fixed;
+  height: 100%;
+  width: 100%;
+  top: 70px;
+  background: rgba(0, 0, 0, 0.2);
+  @media screen and (max-width: 900px ){
+    top: 50px;
+  }
+}
+
+.content {
+  position: relative;
+  width: 500px;
+  height: 660px;
+  background: #FFFFFF;
+  margin: 35px auto;
+  padding: 30px;
```

Comment:
This padding looks a little bit wrong. From the Figma design, it looks like there are 40px of padding on both the left and right sides and 24px of padding on the bottom.